Makefile

```diff
@@ -5,7 +5,7 @@ LDFLAGS=-ldflags "-s -w -X main.Version=${VERSION}"
 TEST=$(shell go list ./... | grep -v /onnx/)
 
 BUILD_PARAMS=CGO_ENABLED=0
-GO1.19=ASSUME_NO_MOVING_GC_UNSAFE_RISK_IT_WITH=go1.19
```

Comment:
You can add this param to `BUILD_PARAMS` like so 
BUILD_PARAMS=CGO_ENABLED=0 ASSUME_NO_MOVING_GC_UNSAFE_RISK_IT_WITH=go1.21

then we also disable CGO for the rest of the commands